servicebus/azure-spring-cloud-starter-servicebus/servicebus-integration/README.md

```diff
@@ -53,7 +53,9 @@ Running this sample will be charged by Azure. You can check the usage and bill a
    and update topic name and subscription in
    [TopicReceiveController.java][topic-receive-controller] and
    [TopicSendController.java][topic-send-controller].
-   
+
+> **NOTE:** When the `ServiceBusMessageHeaders.SESSION_ID` is set in the message headers, and a different `ServiceBusMessageHeaders.PARTITION_KEY` (or `AzureHeaders.PARTITION_KEY`) header is also set,
+> the value of the session id will eventually be used to overwrite the value of the partition key.
```

Comment:
When xxA and xxB is set at the same time, xxx will be used.  

And for what places? should be more clear.  give some refers will be better.